cmd/ff-proxy/main.go

```diff
@@ -370,14 +373,34 @@ func main() {
 			sseClient,
 			messageHandler,
 			stream.WithOnDisconnect(func() {
+				logger.Info("disconnected from Harness SaaS SSE Stream")
+
+				// Set to false so the ProxyService will reject any /stream requests from SDKs until we've reconnected
+				_ = sassToProxyStreamHealth.SetUnhealthy(ctx)
```

Comment:
probably never gonna be an issue, but should we log the error here? just incase?